pyacq/core/stream.py

```diff
@@ -284,10 +309,14 @@ def compress_blosclz4(self, index, data):
         data = blosc.pack_array(data, cname='lz4')
         return index, data
     
-    def send(self, index, data):
-        # TODO deal when data is nparray and self.copy is False and a.flags['C_CONTIGUOUS']=False
-        for f in self.funcs:
-            index, data = f(index, data)
+    def send(self, index, data, autoswapaxes = True):
+        if isinstance(data, np.ndarray):
+            if autoswapaxes and self.params['timeaxis']!=0:
+                data = data.swapaxes(0, self.params['timeaxis'])
+            if not data.flags['C_CONTIGUOUS']:
+                data = data.copy()
```

Comment:
F_CONTINOUS axis=0 is the same as C_CONTINUOUS axis=1 so I think we have the necessary for tunning performences.